### PR TITLE
hdfs upgrade and README File

### PR DESCRIPTION
**Pull request by dheren-git**
Signed-of-by:Dherendra Singh<dhersing@in.ibm.com>

@rajan-mis @mamuthiah

### CODE REVIEW

File: roles/scale_hdfs/upgrade/tasks/upgrade_local_pkg.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+     debuginfo_package: "{{ debuginfo_package + [ item.path ] }}"
+    with_items:
+    - "{{ scale_install_gpfs_hdfs_debuginfo.files }}"
+
+  - name: remove debuginfo from packages
+    set_fact:
+     scale_upgrade_all_packages: "{{ scale_upgrade_all_packages | difference(debuginfo_package)}}"
+  when: not install_debuginfo|bool and ansible_distribution in scale_rhel_distribution
+
+- debug:
```
Comment (rajan-mis) [inline roles/scale_hdfs/upgrade/tasks/upgrade_local_pkg.yml:173]: Can we remove this debug message.

File: roles/scale_hdfs/upgrade/tasks/upgrade_remote_pkg.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+  - name: upgrade | Add GPFS hdfs package to list
+    vars:
+     current_package:  "{{ item.path }}"
+    set_fact:
+     scale_upgrade_all_packages: "{{ scale_upgrade_all_packages + [ current_package ] }}"
+    with_items:
+    - "{{ scale_install_gpfs_hdfs.files }}"
+
+- debug:
```
Comment (rajan-mis) [inline roles/scale_hdfs/upgrade/tasks/upgrade_remote_pkg.yml:120]: Can we remove this debug message.

File: roles/scale_hdfs/upgrade/tasks/zypper/upgrade.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- name: upgrade | Upgrade GPFS HDFS packages
+  zypper:
+   name: "{{ scale_upgrade_all_packages }}"
+   state: latest
+   disable_gpg_check: no
+  when: ansible_fqdn in scale_hdfs_node_list
```
Comment (rajan-mis) [inline roles/scale_hdfs/upgrade/tasks/zypper/upgrade.yml:7]: I think, we may need to remove this when condition.

File: roles/scale_hdfs/upgrade/tasks/upgrade_local_pkg.yml (hunk 1 above)
Comment (dheren-git) [reply roles/scale_hdfs/upgrade/tasks/upgrade_local_pkg.yml]: done

File: roles/scale_hdfs/upgrade/tasks/upgrade_remote_pkg.yml (hunk 2 above)
Comment (dheren-git) [reply roles/scale_hdfs/upgrade/tasks/upgrade_remote_pkg.yml]: done

File: roles/scale_hdfs/upgrade/tasks/zypper/upgrade.yml (hunk 3 above)
Comment (dheren-git) [reply roles/scale_hdfs/upgrade/tasks/zypper/upgrade.yml]: done

**Review by rajan-mis — APPROVED** [review]
Looks good to me @dheren-git

**Pull request merged**